hs/Server.hs

```diff
@@ -58,6 +63,9 @@ runServer = do
               dir "depts" $ liftIO deptList,
               dir "timesearch" searchResponse,
               dir "calendar" $ lookCookieValue "selected-lectures" >>= calendarResponse,
+              -- dir "draw-json?gid=num" $ look "gid" >>= drawJsonResponse,
```

Comment:
Yeah, I just read that section. I can see why comments like that can become permanent "residue" :-)